examples/samples/session.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  generateUuid,
+  QueueClient,
+  SendableMessageInfo,
+  OnSessionMessage,
+  OnError,
+  MessagingError,
+  delay,
+  ServiceBusMessage,
+  ReceiveMode,
+  Namespace,
+  MessageSession
+} from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const path = process.env.QUEUE_NAME || "";
+console.log("str: ", str);
+console.log("path: ", path);
+
+let ns: Namespace;
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(str);
+  const client = ns.createQueueClient(path, { receiveMode: ReceiveMode.peekLock });
+
+  await sendMessages(client, "session-1");
+  await sendMessages(client, "session-2");
+  await sendMessages(client, "session-3");
+  await sendMessages(client, "session-4");
+
+  await receiveMessages(client);
+
+  await client.close();
+}
+
+async function sendMessages(queueClient: QueueClient, sessionId: string): Promise<void> {
+  const data = [
+    { step: 1, title: "Shop" },
+    { step: 2, title: "Unpack" },
+    { step: 3, title: "Prepare" },
+    { step: 4, title: "Cook" },
+    { step: 5, title: "Eat" }
+  ];
+
+  for (let index = 0; index < data.length; index++) {
+    const element = data[index];
+    const message: SendableMessageInfo = {
+      sessionId: sessionId,
+      body: `${element.step} ${element.title}`,
+      label: "RecipeStep",
+      timeToLive: 2 * 60 * 1000, // 2 minutes
+      messageId: generateUuid()
+    };
+
+    console.log(`Message sent: ${message.body} SessionId : ${sessionId}`);
+    await queueClient.send(message);
+  }
+}
+
+async function receiveMessages(queueClient: QueueClient): Promise<void> {
+  const onMessage: OnSessionMessage = async (
+    messageSession: MessageSession,
+    brokeredMessage: ServiceBusMessage
+  ) => {
+    console.log(
+      `Message received: ${brokeredMessage.body} SessionId : ${brokeredMessage.sessionId}`
+    );
+    await brokeredMessage.complete();
+  };
+  const onError: OnError = (err: MessagingError | Error) => {
+    console.log(">>>>> Error occurred: ", err);
+  };
+  queueClient.receiveMessgesFromSessions(onMessage, onError);
```

Comment:
No, we don't have the specific sample for getting messages from specific session but I can change this sample to call receive method for the specific session or we can make one more sample for getting messages from a specific session.
@ramya-rao-a what do you suggest ?

Reply:
I believe it would be useful to show all both ways of receiving
- Receive from a particular session (pick one of the sessions)
- Receive from all sessions (this will only get messages from rest of the sessions)

This way, the user can decide which way they want to go based on their use case

But this also goes against the "have minimal code in samples" argument

Reply:
we can make one more sample specific for getting messages from a specific session, in this way we will have both type of examples in simpler way for the users.